tests/unit/dataactvalidator/test_b12_award_financial_2.py

```diff
@@ -11,22 +11,14 @@ def test_column_headers(database):
     assert (actual & expected_subset) == expected_subset
 
 
-def test_success(database):
+@pytest.mark.parametrize('af', [AwardFinancialFactory(by_direct_reimbursable_fun=None),
+                                AwardFinancialFactory(by_direct_reimbursable_fun=''),
+                                AwardFinancialFactory(by_direct_reimbursable_fun='R'),
+                                AwardFinancialFactory(by_direct_reimbursable_fun='r'),
+                                AwardFinancialFactory(by_direct_reimbursable_fun='D'),
+                                AwardFinancialFactory(by_direct_reimbursable_fun='d')])
```

Comment:
I like parametrize here, but wonder about creating whole objects -- we want to be as concise as possible. Would it make sense to have the parameter just be `[None, '', 'R', 'r'...]` and then create the `AwardFinancial` in the test?